src/py/flwr/common/logger.py

```diff
@@ -303,6 +305,12 @@ def restore_output() -> None:
     console_handler.stream = sys.stdout
 
 
+def redirect_output(output_buffer: StringIO) -> None:
+    """Redirect stdout and stderr to text I/O buffer."""
+    sys.stdout = output_buffer
+    sys.stderr = output_buffer
```

Comment:
```suggestion
def redirect_output(output_buffer: StringIO) -> None:
    """Redirect stdout and stderr to text I/O buffer."""
    sys.stdout = output_buffer
    sys.stderr = output_buffer
    console_handler.stream = sys.stdout
```

Reply:
@panh99  Since we aren't redirecting any logs for `flwr ls` and `flwr run`, we shouldn't need to redirect `console_handler.stream = sys.stdout`?

Reply:
It's not needed for now, but to maintain consistency and avoid potential confusion in the future, I recommend also redirecting logs, as they are also part of the "output."

Reply:
Hmm ok. I'm leaning against it since it's unnecessary for now, but will add anyway.

Reply:
Thanks. I think otherwise we have to clarify in the docstring that logs may or may not be redirected, depending on when the `log` module is imported.